bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/protobuf/BigtableZeroCopyByteStringUtil.java

```diff
@@ -12,6 +12,29 @@
  * @version $Id: $Id
  */
 public final class BigtableZeroCopyByteStringUtil {
+
```

Comment:
It does indeed break in protubuf 3.0.0; LiteralByteString is now a private class, and this code doesn't work anymore.

FYI [here's](https://github.com/apache/hbase/blob/master/hbase-protocol/src/main/java/com/google/protobuf/HBaseZeroCopyByteString.java#L67) the grandaddy of both our copy and the asynchbase code.